roles/rabbitmq/defaults/main.yml

```diff
@@ -4,6 +4,8 @@ rabbitmq_server: localhost
 rabbitmq_interface:
   - 127.0.0.1
   - ::1
+  - "{% if ansible_default_ipv4 %}{{ ansible_default_ipv4.address }}{% endif %}"
+  - "{% if ansible_default_ipv6 %}{{ ansible_default_ipv6.address }}{% endif %}"
```

Comment:
While users can specify their interfaces, defaulting to `0.0.0.0` seems reasonable to me.